### PR TITLE
Feat/remove css interop sra1

### PR DESCRIPTION
**Pull request by sra1kumar-NULL**
Removed cssInterop from 
  1. radio  
  2. checkbox  
  3. form-control  
  4. input  
  5. avatar  
  6. image  
  7. accordion  
  8. textarea

### CODE REVIEW

File: example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx [hunk 1]
```diff
@@ -171,11 +171,6 @@ const UIRadio = createRadio({
   Label: LabelWrapper,
 });
 
-cssInterop(UIRadio, { className: 'style' });
-cssInterop(UIRadio.Group, { className: 'style' });
-cssInterop(IndicatorWrapper, { className: 'style' });
```
Comment (Viraj-10) [inline example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx:176]: Also remove wrappers around it. Don't need those as well.

File: example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx [hunk 2]
```diff
@@ -171,11 +171,6 @@ const UIRadio = createRadio({
...
-cssInterop(LabelWrapper, { className: 'style' });
-
 cssInterop(IconWrapper, {
```
Comment (Viraj-10) [inline example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx:174]: Remove `IconWrapper` directly use `PrimitiveIcon` directly.

File: example/storybook-nativewind/src/core-components/nativewind/textarea/index.tsx [hunk 3]
```diff
@@ -23,7 +23,6 @@ const UITextarea = createTextarea({
   Input: TextInput,
 });
 
-cssInterop(TextareaWrapper, { className: 'style' });
 cssInterop(UITextarea.Input, {
```
Comment (Viraj-10) [inline example/storybook-nativewind/src/core-components/nativewind/textarea/index.tsx:26]: Check if we need to handle this or not. if `TextInput already has it then remove this as well`

**Review by Viraj-10 — CHANGES_REQUESTED** [review]

File: example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx (hunk 1 above)
Comment (sra1kumar-NULL) [reply example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx:176]: Done, Removed the wrappers.

File: example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx (hunk 2 above)
Comment (sra1kumar-NULL) [reply example/storybook-nativewind/src/core-components/nativewind/radio/index.tsx]: Replaced IconWrapper with PrimitiveIcon directly.

File: example/storybook-nativewind/src/core-components/nativewind/textarea/index.tsx (hunk 3 above)
Comment (sra1kumar-NULL) [reply example/storybook-nativewind/src/core-components/nativewind/textarea/index.tsx]: Tested TextInput behaviour without any nativeToStyle props for textAlign in all platforms working fine. So removed the css Interop

**Review by Viraj-10 — APPROVED** [review]

**Pull request merged**